src/mono/wasm/runtime-test.js

```diff
@@ -102,51 +102,56 @@ if (typeof globalThis.performance === 'undefined') {
     }
 }
 
-var Module = {
-    no_global_exports: true,
-    mainScriptUrlOrBlob: "dotnet.js",
-    config: null,
-    configSrc: "./mono-config.json",
-    print: console.log,
-    printErr: console.error,
-    onConfigLoaded: function () {
-        if (!Module.config) {
-            console.error("Could not find ./mono-config.json. Cancelling run");
-            fail_exec(1);
-        }
-        // Have to set env vars here to enable setting MONO_LOG_LEVEL etc.
-        for (let variable in processedArguments.setenv) {
-            Module.config.environment_variables[variable] = processedArguments.setenv[variable];
-        }
+loadDotnet("./dotnet.js").then((createDotnetRuntime) => {
+    return createDotnetRuntime(({ MONO, INTERNAL, BINDING, Module }) => ({
+        disableDotNet6Compatibility: true,
+        mainScriptUrlOrBlob: "dotnet.js",
```

Comment:
should this be ```./dotnet.js``` for consistency with mono-config below?

Reply:
this was some legacy thing deleted.